content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
-:guilabel:`Paid Time Off Allocation`, :guilabel:`273S Sheet`, and :guilabel:`274.XX Sheets` reports
-are specific to Belgian companies only.
+The :guilabel:`Reporting` section of the *Payroll* app offers a variety of reports to choose from,
+organized by location.
 
-To view a report, go to :menuselection:`Payroll --> Reporting` and click on the specific report.
+The *Payroll* report, *Work Entry Analysis* report, and *Salary Attachment Report* are default
+reports in the *Payroll* app, and are available for all companies, regardless of location.
+
+Beneath the three default reports are all localization-based reports, organized by country, in
+alphabetical order. These reports contain all the various information for the offered benefits and
+local tax laws.
+
+To view all the available reports for the database, including all the localization-specific ones,
+navigate to :menuselection:`Payroll app --> Reporting` to view the available reports. Click on a
+specific report to view it.
 
 .. image:: reporting/reports.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Report dashboard view showing extra reports for Belgium databases.
 
-If a report is unavailable to a company, a user error pops up, stating `You must be logged in to a
-Belgian company to use this feature.`
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: `You must be logged in to a (country) company to use this feature.`
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Filters
+=======
 
-.. image:: reporting/custom-filter.png
-   :align: center
-   :alt: Report dashboard view.
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+Click the :guilabel:`drop-down arrow` in the search box to display the available
+:guilabel:`filters`. Filters show information that matches the specific filter parameters.
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the :guilabel:`Current
+   month:(Month)(Year)` filter, and the :guilabel:`Validated` filter. The :guilabel:`Payroll` report
+   only has one default filter, the :guilabel:`Last 365 Days Payslip` filter.
```

Comment:
is there a reason you didn't mention the default filter in the 'Salary Attachment Report'? could be useful...

Reply:
No- I was just illustrating that one report has only one filter, and a different one has two. It wasn't to intentionally leave out the third report- but now that you point it out, I see why it could be confusing. Adding that info!